ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -952,6 +954,24 @@ private class JdbcLedgerDao(
           Try {
             storeTransaction(offset, tx, txBytes)
 
+            tx.transaction.roots.iterator
+              .map(nid => split(nid).fold(sys.error(s"########## can't split $nid"))(_.nodeId))
```

Comment:
Can we have a more meaningful error here?

Reply:
That's actually a leftover from debugging, I'll remove it, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5029/commits/e2b82530893ccb82d932fdcc5198ac0ccd829928